refinery/file_store/migrations/0003_filetypes_and_fileextensions.py

```diff
@@ -0,0 +1,378 @@
+        Filetype(
+            description="FASTA file",
+            name="FASTA"),
+        Filetype(
+            description="FASTQ file",
+            name="FASTQ"),
+        Filetype(
+            description="FASTQC Sanger",
+            name="FASTQCSANGER"),
+        Filetype(
+            description="FASTQ Illumina",
+            name="FASTQILLUMINA"),
+        Filetype(
+            description="FASTQ Sanger",
+            name="FASTQSANGER"),
+        Filetype(
+            description="FASTQ Solexa",
+            name="FASTQSOLEXA"),
+        Filetype(
+            description="Portable Document Format",
+            name="PDF"),
+        Filetype(
+            description="Sequence Alignment/Map",
+            name="SAM"),
+        Filetype(
+            description="Segmented Data File",
+            name="SEG",
+            used_for_visualization=True),
+        Filetype(
+            description="Tabular file",
+            name="TABULAR"),
+        Filetype(
+            description="TDF file",
+            name="TDF",
+            used_for_visualization=True),
+        Filetype(
+            description="Gzip compressed tar archive",
+            name="TGZ"),
+        Filetype(
+            description="Text file",
+            name="TXT"),
+        Filetype(
+            description="Variant Call Format",
+            name="VCF",
+            used_for_visualization=True),
+        Filetype(
+            description="Wiggle Track Format",
+            name="WIG",
+            used_for_visualization=True),
+        Filetype(
+            description="XML file",
+            name="XML"),
+        Filetype(
+            description="Zip compressed archive",
+            name="ZIP"),
+        Filetype(
+            description="FASTQ.GZ file",
+            name="FASTQ.GZ"),
+        Filetype(
+            description="GCT file",
+            name="GCT"),
+        Filetype(
+            description="DAT file",
+            name="DAT"),
+        Filetype(
+            description="Illumina basecall file",
+            name="BCL"),
+        Filetype(
+            description="Illumina basecall file gzipped",
+            name="BCL GZip file"),
+        Filetype(
+            description="Illumina basecall file tarred + gzipped",
+            name="BCL TGZ file"),
+        Filetype(
+            description="BAM index file",
+            name="BAI file",
+            used_for_visualization=True),
+        Filetype(
+            description="Cooler file",
+            name="COOLER",
+            used_for_visualization=True
+        ),
+        Filetype(
+            description="HITILE file",
+            name="HITILE",
+            used_for_visualization=True
+        ),
+        Filetype(
+            description="BEDGRAPH file",
+            name="BEDGRAPH",
+            used_for_visualization=True
+        )
+    ]
+
+    existing_filetypes = [filetype.name for filetype in Filetype.objects.all()]
+
+    for filetype in filetypes:
```

Comment:
Why not simply use [bulk_create](https://docs.djangoproject.com/en/1.7/ref/models/querysets/#bulk-create) and pass on IntegrityError to avoid duplicates?